conf/messages.en

```diff
@@ -191,12 +191,15 @@ authenticate.password = Password
 authenticate.submit = Submit
 authenticate.new = Are you new? <a href="#" id="form-open-sign-up">Sign up!</a>
 authenticate.non.member = Not a member? <a href="/signUp">Sign up now</a>
+authenticate.non.member.mobile = Not a member? <a href="/signUpMobile">Sign up now</a>
```

Comment:
Not sure what you mean here as well, the reason it is different is because of routing:
`<a href="/signUpMobile">`

Reply:
Ahh, my bad. I was just thinking about the signing up through the navbar and forgot about the other actual sign in/up pages. So the wording is obviously good, but could you change the translations to take in the link as a parameter? So I think the translation would probably look like:
```
authenticate.non.member.mobile = Not a member? <a href="{0}">Sign up now</a>
```

Then you would pass in the URL in the scala.html. There are examples elsewhere in the code you can go off of.

Then of course do that for both the sign in and sign up translations.

Reply:
Fixed this issue. This shows up a lot throughout the code so if you want me to go through and fix every occurrences let me know.

Reply:
Oh, yes please! That's kind of what I meant. Since the only difference between the mobile and desktop translations is the link, you could have just a single translation if the link is always passed in as a parameter.